src/main/scala/org/broadinstitute/clio/ClioServer.scala

```diff
@@ -32,7 +33,7 @@ object ClioServer
   private val wrapperDirectives: Directive0 = {
     auditRequest & auditResult & completeWithInternalErrorJson & auditException & mapRejectionsToJson
   }
-  private val innerRoutes: Route = concat(statusRoutes)
+  private val innerRoutes: Route = concat(authorizationRoute, statusRoutes)
```

Comment:
The /authorization route is there for testing.
It isn't even exported as an endpoint yet --
though more by default than by design.